packages/web-components/src/components/ic-side-navigation/ic-side-navigation.stories.mdx

```diff
@@ -66,7 +66,7 @@ import readme from "./readme.md";
             </svg>
           </ic-navigation-item>
           <ic-divider slot="primary-navigation"></ic-divider>
-          <ic-navigation-item slot="primary-navigation" href="/" label="Trends">
+          <ic-navigation-item slot="primary-navigation" label="Trends">
```

Comment:
Not sure if we need an example showing it works without an href? I think it might be ok if we just test this locally - just thinking if we come back to this story in future we might wonder why it doesn't have an href, or if we're demoing a change to the side nav and then this link doesn't work or something

But only a tiny thing 🤷‍♀️